src/components/buttons/button.tsx

```diff
@@ -1,11 +1,21 @@
 import React from "react"
 
-const button = () => {
+interface PropTypes {
```

Comment:
It's better to rename this to `ButtonProps` than proptypes. Proptypes convey a different meaning in my opinion 
```suggestion
interface ButtonTypes {
```